src/qibotn/backends/cutensornet.py

```diff
@@ -1,19 +1,9 @@
-import cuquantum  # pylint: disable=import-error
```

Comment:
Hi @BrunoLiegiBastonLiegi and @scarrazza , this import is necessary, else line71 will fail. (        self.cuquantum = cuquantum)

Reply:
I believe you meant L61:
https://github.com/qiboteam/qibotn/blob/217a6338bce0120159fc77648ec38d8d894197c2/src/qibotn/backends/cutensornet.py#L61
but the import could be moved close to
https://github.com/qiboteam/qibotn/blob/217a6338bce0120159fc77648ec38d8d894197c2/src/qibotn/backends/cutensornet.py#L15
for the same reason.

Reply:
Yes L61. 
Do you mean to say to move the "import cuquantum" close to line 15?
If so, I am also ok with that suggestion, just that without any import, the code will not run.